99.md

```diff
@@ -0,0 +1,170 @@
+# NIP-99
+
+## Prediction Markets
+
+`draft` `optional` `author:ekzyis`
+
+This NIP specifies how prediction markets can be implemented using nostr and lightning.
+
+Any user ("the creator") can create a _predictable event_ (`kind 1099`) such that other users ("the participants") can buy and sell shares of possible outcomes using zap requests as defined by NIP-57 and reverse zap requests (TBD).
```

Comment:
With "reverse zap requests" I mean an event you create and the other party does zap.

Basically my idea to implement something like LNURL-withdraw on nostr